wordpress-mobile/WordPress-iOS
# Comment Reply: update minimized UI

Ref: #17412 

This updates the UI for the minimized Reply view.

Notes:
- The text size does not update on the fly (this is legacy).
- Placeholder text does not yet match design.
- The fullscreen view is not updated yet.

To test:
- Go to any of these views as they all use the same Reply view:
  - My Site > old Comment details.
  - My Site > new Comment details.
  - Notifications > Comments > notification details.
  - Reader > post comments > reply to post or comment.
- Verify the `Reply` button is not shown initially.
- Enter some text. Verify the `Reply` button is shown.
- Enter some returns. Verify both buttons are constrained to the top.

| ![Simulator Screen Shot - iPhone 12 Pro Max - 2021-11-09 at 17 18 01](https://user-images.githubusercontent.com/1816888/141026516-f233734a-ee84-4b3a-bb13-01970eced549.png) | ![Simulator Screen Shot - iPhone 12 Pro Max - 2021-11-09 at 17 18 06](https://user-images.githubusercontent.com/1816888/141026549-516248da-9b8a-43a0-b17c-438c9685e5a3.png) | ![Simulator Screen Shot - iPhone 12 Pro Max - 2021-11-09 at 17 18 10](https://user-images.githubusercontent.com/1816888/141026567-a1800f09-b346-4dba-b3a7-22dc27a6dbe4.png) |
|--------|-------|-------|

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Verified the UI looked and functioned correctly.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
